app/js/containers/feedbackify_container.js

```diff
@@ -1,21 +1,13 @@
 import { connect } from 'react-redux'
 import Feedbackify from '../components/feedbackify/feedbackify'
-import { openFeedbackDialog } from '../actions/feedbackify'
 
-const mapStateToProps = (state) => {
-  // There are no props to map, but we need a function to call connect()!
-  return {}
-}
-
-const mapDispatchToProps = (dispatch) => {
-  return {
-    feedbackButtonClick: () => {
-      dispatch(openFeedbackDialog())
-    },
+const mapStateToProps = ({analyticsApp}) => {
+  if (analyticsApp.profile) {
+    fby.push(['setEmail', analyticsApp.profile.email]);
   }
+  return {}
 }
```

Comment:
This should be just:

```
const mapStateToProps = ({analyticsApp}) => ({ email: analyticsApp.profile.email })
```

and the `onClick` method of the Feedbackify component is the one that should do both push, because it will be receiving `email` inside `this.props`.